charts/consul/templates/crd-gatewayclassconfigs.yaml

```diff
@@ -39,163 +51,116 @@ spec:
           metadata:
             type: object
           spec:
-            description: Spec defines the desired state of GatewayClassConfig.
+            description: This is a Resource type.
             properties:
+              consul:
+                properties:
+                  address:
+                    type: string
+                  authentication:
+                    properties:
+                      address:
+                        type: string
+                      managed:
+                        type: boolean
+                      method:
+                        type: string
+                      namespace:
+                        type: string
+                    type: object
+                  ports:
+                    properties:
+                      grpc:
+                        format: int32
+                        type: integer
+                      http:
+                        format: int32
+                        type: integer
+                    type: object
+                  scheme:
+                    type: string
+                type: object
               copyAnnotations:
-                description: Annotation Information to copy to services or deployments
                 properties:
                   service:
-                    description: List of annotations to copy to the gateway service.
                     items:
                       type: string
                     type: array
                 type: object
               deployment:
-                description: Deployment defines the deployment configuration for the
```

Comment:
We expect the GatewayClassConfig to change pretty quickly (want to rework it to something more generic), so the missing descriptions are fine.